### PR TITLE
Create basic UI

### PR DESCRIPTION
**Pull request by stephnna**
Hi code reviewer,

I implemented the following
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.

Kindly review as neccessary

### CODE REVIEW

File: main [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                6 - List all rentals for a given person id
+                7 - Exit
+              ")
+      start
+    end
+  end
+end
+
+main = Main.new
+main.run
```
Comment (Bondok6) [inline main]: - [x] I think you forgot to add the extension ".rb", the file name should be "main.rb" 💯

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @stephnna 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Your README is professional ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  end
+
+  def list_rentals
+    id = take_input_label('Enter id of person')
+    @rentals.each do |rental|
+      put_lable('Rentals')
+      rental.person.id && id
+      put_lable("#{rental.date} - #{rental.book.title}")
+    end
+  end
```
Comment (Bondok6) [inline app.rb]: Great job, everything works fine 💯 except for the `list_rentals` method when I try to enter a person's id it gives me an error, so Kindly fixt that 👍

https://user-images.githubusercontent.com/45231276/193787639-f1e25944-80e3-4998-a7e1-830b750773d4.mp4

File: main (hunk 1 above)
Comment (stephnna) [reply main]: DONE

File: app.rb (hunk 2 above)
Comment (stephnna) [reply app.rb]: Done

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @stephnna👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**